<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Core</class>
 <widget class="QMainWindow" name="Core">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Core</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resourcen.qrc">
    <normaloff>:/Stock/data/icon.png</normaloff>:/Stock/data/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>386</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kamera Import Ordner</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="path_kamera_folder">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>30</y>
      <width>526</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="Upload_Radio">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>5</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_BTN">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>55</y>
      <width>386</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Ornder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="upload_folder">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>75</y>
      <width>526</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="prozent_werbung">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>260</y>
      <width>96</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>235</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>prozent_werbung</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prozent_kleines_bild">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>260</y>
      <width>106</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>235</y>
      <width>116</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>prozent_kleines_bild</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>235</y>
      <width>116</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>prozent_grosses_bild</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prozent_grosses_bild">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>260</y>
      <width>106</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="background">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>260</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>235</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hintergrund?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_viewer_BTN">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>290</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>start Viewer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="viewer_path">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>120</y>
      <width>526</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>100</y>
      <width>386</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>viewer.exe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="upload_path">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>210</y>
      <width>526</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>190</y>
      <width>386</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FTP-Upload.exe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Upload_folder_name">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>290</y>
      <width>176</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>295</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Ordner</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="full_size_folder">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>165</y>
      <width>526</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>145</y>
      <width>386</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Original Qualit√§t Ordner</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="compressed_width">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>260</y>
      <width>106</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>235</y>
      <width>116</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Comprimieren Breite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_BTN">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>46</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_BTN">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>46</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Resourcen.qrc"/>
 </resources>
 <connections/>
</ui>
